cpp/src/io/json/read_json.cu

```diff
@@ -248,7 +248,8 @@ table_with_metadata read_batch(host_span<std::unique_ptr<datasource>> sources,
   // If input JSON buffer has single quotes and option to normalize single quotes is enabled,
   // invoke pre-processing FST
   if (reader_opts.is_enabled_normalize_single_quotes()) {
-    normalize_single_quotes(bufview, stream, cudf::get_current_device_resource_ref());
+    normalize_single_quotes(
+      bufview, reader_opts.get_delimiter(), stream, cudf::get_current_device_resource_ref());
```

Comment:
Just for verification. The `get_delimiter()` will return `\n` if the caller did not set a delimiter in the `json_reader_options`?

Reply:
Yes, the default value set in `json_reader_options` for the delimiter is `\n` (https://github.com/rapidsai/cudf/blob/150d8d8c3a3a4aec87004fd0d130b56e388fa43d/cpp/include/cudf/io/json.hpp#L116)